.github/workflows/lint-test.yaml

```diff
@@ -129,9 +131,50 @@ jobs:
         uses: helm/kind-action@v1.10.0
         if: steps.list-changed.outputs.changed == 'true'
 
-      - name: Run chart-testing (install)
+      - name: Run chart-testing (install with postgresql.enabled)
         id: install
         if: steps.list-changed.outputs.changed == 'true'
         run: |
             ct install --target-branch ${{ github.event.repository.default_branch }} \
             --helm-extra-set-args "--set=postgresql.enabled=true --set=postgresql.global.postgresql.auth.password=testing123456 --set=internalDatabase.enabled=false --set=externalDatabase.enabled=True --set=externalDatabase.type=postgresql --set=externalDatabase.password=testing123456"
+
+  test-hpa:
+    runs-on: ubuntu-22.04
+    needs: [changes, lint]
+    if: needs.changes.outputs.src != 'false' && github.event.label.name == 'hpa'
```

Comment:
I think we should run it all the time, otherwise the label might be missed on a PR and then a bug can slip in unnoticed. Running one extra job is really no concern to me, especially since it doesn't take a long time to complete.

Reply:
Always would be best imo :)